### ISSUE TITLE
Is it possible to use Mask R-CNN when objects of the same instance are in different parts of the image ?

### ISSUE THREAD

**Issue by YuriGagarine** [matterport/Mask_RCNN#2267]
I want to use Mask R-CNN to make instance segmentation on spectrogram images. As you may know, signals comming from the same source can make patterns in different part of the spectrogram simply because the source can emmit at different time.

My question is: do you think that Mask R-CNN be able to treat such an instance-segmentation problem, that is to say to make detection and be able to set the same instance number for several region in the image ?

**Comment by konstantin-frolov** [comment]
Yes it is possible. Only I recommend that you set the first value in RPN_ANCHOR_SCALES to Config ~ The size of the signal area in the spectrogram.
